about/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy  | Doodle Dandy</title>
+    <link rel="stylesheet" href="../assets/vendor/simplecss/simple.min.css" />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/fontawesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/brands.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/solid.min.css"
+    />
+    <link rel="stylesheet" href="../assets/styles/main.css" />
+    <link
+      rel="shortcut icon"
+      href="../assets/images/favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
+
+  <body>
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/contact" >Contact us</a>
+        <a href="/about" class="current">About us</a>
+      </nav>
+    </header>
+    <main>
+      <h1>About us</h1>
+      <p>
+        Our mission is to raise healthy, happy, and well-socialized pups so you can experience the joy of owning a Doodle. Every puppy is raised in our home—none of our dogs, including moms, pups, and papa dogs, live in a kennel. These dogs have brought immense happiness into our lives, and it's a privilege to share that love and companionship with others through responsible breeding. Let us help you bring your next family member home—you'll love how smart and loyal these AussieDoodles are! We invite you to join our Doodle Dandy Extend-A-Doodle family!
```

Comment:
You're only doing `AussieDoodles`? What does `Extend-A-Doodle` mean?

You should have a space before and after the dashes in the two sentences that contain them.